src/main/java/org/itsallcode/openfasttrace/importer/legacytag/LegacyTagImporterFactory.java

```diff
@@ -39,43 +40,61 @@
 
     private static LegacyTagImporterConfig defaultConfig = new LegacyTagImporterConfig();
 
-    private final List<PathConfig> pathConfigs;
+    private final Supplier<LegacyTagImporterConfig> config;
 
     public LegacyTagImporterFactory()
     {
-        this(defaultConfig.getPathConfigs());
+        this(() -> defaultConfig);
     }
 
-    public LegacyTagImporterFactory(final List<PathConfig> pathConfigs)
+    public LegacyTagImporterFactory(final LegacyTagImporterConfig config)
     {
-        this.pathConfigs = pathConfigs;
+        this(() -> config);
+    }
+
+    public LegacyTagImporterFactory(final Supplier<LegacyTagImporterConfig> config)
+    {
+        this.config = config;
     }
 
     @Override
-    public boolean supportsFile(final Path file)
+    public boolean supportsFile(final Path path)
     {
-        return findConfig(file).isPresent();
+        return findConfig(path).isPresent();
     }
 
     private Optional<PathConfig> findConfig(final Path file)
     {
-        return this.pathConfigs.stream().filter(config -> config.matches(file)).findFirst();
+        final Path relativePath = makeRelative(file);
+        return this.config.get().getPathConfigs().stream() //
+                .filter(config -> config.matches(relativePath)) //
+                .findFirst();
+    }
+
+    private Path makeRelative(final Path path)
+    {
+        final Path basePath = this.config.get().getBasePath();
+        if (basePath == null)
+        {
+            return path;
+        }
+        return basePath.relativize(path);
```

Comment:
Does the match still work if the paths in the path config are provided absolute?

Reply:
No. I created #127 to check for absolute paths.

Reply:
Perfect. Thanks.